containers/test-apps/kitchen/bin/kitchen

```diff
@@ -121,6 +121,11 @@ def gzip_compress(in_data):
     return out_data
 
 
+def truncate(in_string, truncate_length):
+    "Collapse and truncate the given text to the given length."
+    return in_string[:truncate_length] + (in_string[truncate_length:] and '...')
```

Comment:
I find `(in_string[truncate_length:] and '...')` confusing... Why not `if len(in_string) > truncate_length`? I'd prefer not having the extra slicing operations when that condition fails.

Reply:
Done, uses `if-else` block